charts/flink-operator/templates/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.mutatingWebhook.enabled }}
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ .Release.Name }}-serving-cert
+  name: {{ .Release.Name }}-mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: {{ .Release.Name }}-webhook-service
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-flinkoperator-k8s-io-v1beta1-flinkcluster
+  failurePolicy: Fail
+  name: mflinkcluster.flinkoperator.k8s.io
+  {{- if .Values.namespaceScoped }}
+  namespaceSelector:
```

Comment:
@diranged I think this namespaceSelector is causing issue on the job submitter hanging in running status after submission.

Reply:
The way the operator deploy via the make target from the repo has this extra logic to set the label flink-operator-namespace. https://github.com/spotify/flink-on-k8s-operator/blob/5db74d5ff6562af1b3bd5fc9c70a4f4084000263/Makefile#L98-L103

Reply:
updated hard coded key `flink-operator-namespace` to `kubernetes.io/metadata.name` instead to avoid setting the label. 

Also made change in `flilnk-job` to resolve job submitter hanging and other issues due to not using webhook.

Reply:
awsome!